packages/@functionless/aws-constructs/test/queue.localstack.test.ts

```diff
@@ -273,7 +264,7 @@ runtimeTestSuite<{
       return {
         queue,
         outputs: {
-          busName: addr.node.addr,
+          busName: bus.resource.eventBusName,
```

Comment:
Oops lol. How did that ever work

Reply:
It did work, but the bus was a singleton.